chore: update Python versions and CI

johnfraney opened the pull request:
Updates supported Python versions to be 3.9-3.13 and updates CI to use the built-in `textblob.download_corpora` command. Also updates corpora module names to fix a missing corpora error when running:

python -m textblob.download_corpora

## Notes

This should fix CI errors and https://github.com/sloria/TextBlob/issues/482 and https://github.com/sloria/TextBlob/issues/474 but removes NLTK v3.8 support

This requires updating the repo's required status checks to be Python 3.9 and 3.13

johnfraney committed:
fix: update corpora module names

Updates corpora module names to fix a missing corpora error when
running:

python -m textblob.download_corpora

This should fix CI errors and #482 and #474

johnfraney committed:
chore: update Python versions and CI

Updates supported Python versions to be 3.9-3.13 and updates CI to use
the built-in textblob.download_corpora command

johnfraney committed:
fix: corpora download in CI

sloria commented on .github/workflows/build-release.yml:
```diff
@@ -13,18 +13,17 @@ jobs:
         include:
-          - { name: "3.8", python: "3.8", tox: py38 }
-          - { name: "3.12", python: "3.12", tox: py312 }
-          - { name: "lowest", python: "3.8", tox: py38-lowest }
```
i think we want to keep the `lowest` build, to test against the lowest supported python+nltk version. so this will need to be changed to `tox: py39-lowest`

johnfraney replied:
Ah, okay, that makes sense. I'll get this updated

johnfraney replied:
@sloria Does `lowest` look okay to you now?

sloria replied:
yes 👍

sloria reviewed (changes requested):
thank you! been meaning to get to this for a while but haven't had the time.

once `lowest` is added back, this should be good to merge!

johnfraney committed:
fix: bring back lowest env in Tox/CI

Adds back the "lowest" env in Tox/CI to ensure support in the lowest supported Python + NLTK versions

sloria reviewed (approved):
looks great! thanks so much.

i'll fix up the branch rules and merge this later today. in the meantime, can you add yourself to AUTHORS.rst please?

johnfraney commented:
@sloria Oh! You bet. I'll do that now

johnfraney committed:
chore: add johnfraney to Authors.rst

sloria committed:
Update changelog

sloria committed:
Update changelog
